src/Feature/Account/client/Integration/Account/sagas.ts

```diff
@@ -79,26 +80,19 @@ export function* changePassword(action: Action<ChangePasswordPayload>) {
     return;
   }
 
-  const { payload } = action;
-  const { oldPassword, newPassword } = payload;
-
-  const changePasswordPayload = {
+  const payload: Commerce.ChangePasswordRequest = {
+    ...action.payload,
     email: commerceUser.email,
-    newPassword,
-    oldPassword,
   };
 
   yield put(actions.ChangePasswordRequest());
-  const { data, error }: Result<Commerce.ChangePasswordResultModel> = yield call(
-    Api.changePassword,
-    changePasswordPayload
-  );
+  const { data, error }: Result<boolean> = yield call(Api.changePassword, payload);
 
   if (error) {
     return yield put(actions.ChangePasswordFailure(error.message, error.stack));
   }
 
-  if (!data.passwordChanged) {
+  if (!data) {
```

Comment:
what is data now? why `!data.passwordChanged` condition was removed?

Reply:
because I passed `boolean` value in `data` instead object